crates/typst-cli/src/compile.rs

```diff
@@ -198,6 +198,41 @@ enum ImageExportFormat {
     Svg,
 }
 
+struct OutputTemplate;
+
+impl OutputTemplate {
```

Comment:
please move this below the export code (we always order from general -> specific)